_test-server/views/component-wrapper.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
The main reason was to be able to add a wrapper around the component but I agree would be nicer to use `vanilla` so I am in the process of switching that but it means `vanilla` will have a wrapper added based on a flag